scripts/mover.py

```diff
@@ -17,12 +17,13 @@
     "--cache_mount",
     help="Cache mount point in Unraid. This is used to additionally filter for only torrents that exists on the cache mount."
     "Use this option ONLY if you follow TRaSH Guides folder structure.",
-    default=None,
```

Comment:
Why is cache now required with no default of none?

Reply:
required in order to build correct path, as it is going to use it for moving

Reply:
The fact it is required means `Use this option ONLY if you follow TRaSH Guides folder structure.` is now an invalid comment

Reply:
well, agreed